WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -153,7 +153,7 @@ final class OrdersRootViewController: UIViewController {
     }
 
     func selectOrder(for orderID: Int64) {
-        ordersViewController.selectOrder(for: orderID)
+        ordersViewController.selectOrder(for: orderID, canSwitchDetails: false)
```

Comment:
I'm not _sure_, but we might be able to simplify this when the feature flag is removed. If we can, it'd be good to note that now, becaues I don't think it will be obvious then.

From what I can see in the callers of `OrdersRootViewController.selectOrder(for:)`, it only gets called when the feature flag is false. It's used for some deeplink handling.

I checked whether it will break deeplinks with the flag off... but it's fine because that's handled by `navigationController?.pushViewController(loaderViewController, animated: true)` in `presentDetails`.